WTWSMS/events/ze_lord_spiritual_events.txt

```diff
@@ -29,7 +29,6 @@ character_event = {
 			independent = yes
 			higher_tier_than = DUKE
 		}
-		proselyte_religion_trigger = yes
```

Comment:
Your setup is ahistorical.

There were some prolestyzing Jews, such as the Khazars. There were some prolestyzing Zoroastrians. There were prolestyzing Arians. There were prolestyzing Hindus. There were _many_ prolestyzing Buddhists.

All of these are barred because you railroaded through an unnecessary restriction in that one Event.
Also, the LI installation you made is NOT a core feature of the Mod, and frankly, players should be allowed to opt out of it by choosing it through a Rule. Some people like dynamic religions.